setup.py

```diff
@@ -10,7 +10,8 @@
     "pytest-flake8",
     "pytest-coverage",
     "asynctest",
-    "black",
+    "black==19.10b0",
```

Comment:
Is black with this particular version still needed as a requirement? If not then please remove it.

Reply:
Yep, its still required.